Sources/HeliumLogger/HeliumLogger.swift

```diff
@@ -61,16 +61,133 @@ public class HeliumLogger {
     /// If true, use the detailed format when a user logging format wasn't specified.
     public var details: Bool = true
 
+    /// If true, use the full file path, not just the filename.
+    public var fullFilePath: Bool = false
```

Comment:
Would it be better to just add `(%fullfile)` to `HeliumLoggerFormatValues` rather than having this separate state in the system?